modules/KIWILinuxRC.sh

```diff
@@ -802,11 +803,17 @@ function udevStart {
             enablePlymouth=0
             break
             ;;
+        case "$o" in
+            fips=1*)
+            fipsEnabled=1
+            ;;
         esac
     done
     if [ $enablePlymouth -eq 1 ]; then
         startPlymouth
     fi
+    # start haveged
+    startHaveged
```

Comment:
Well, haveged is used also without fips mode enabled. But for me, the change is fine.